### PR TITLE
feat(Landing page): setup the frontend boilerplate and implemented the landing page

### PR DESCRIPTION
**Pull request by Cyebukayire**
#2 Frontend UI/UX

## Task Completed:

- [x]  Designed UI elements that are a must-have (ie, header, search elements, calendar design to display rides, etc)
- [x]  Used Figma to represent the sketched wireframes.
- [x]  Refactored the Frontend Setup (Re-arranged project structure for easy collaboration).
- [x]  Implemented Landing Page (From the mockup)
- [x]  Responsiveness
Used: SCSS (To be able to use constant CSS variables that will be reusable for all components) & JSX

##  A screenshot of the Landing Page
image

### CODE REVIEW

**Comment by andrewtlu** [comment]
Did you download any other packages? I can't get `npm start` to run locally.

Also, [navbar](https://forms.gle/aQNgCE8bWz6WQFaPA) should be renamed to Navbar if I'm not mistaken.

**Comment by Cyebukayire** [comment]
> Did you download any other packages? I can't get `npm start` to run locally.
> 
> Also, [navbar](https://forms.gle/aQNgCE8bWz6WQFaPA) should be renamed to Navbar if I'm not mistaken.

Ohh Thank you.
Are you running `npm start` from the **frontend** directory?
What's the message you get after running npm start, I installed a couple more dependencies but the `scripts` to run the program were not changed.

File: frontend/src/components/Services/services.scss [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+        padding-right: 0;
+        margin: 0;
+    }
+    .services .services-section-2{
+        flex-direction: column;
+        justify-content: center;
+    }
+}
+
+@media (max-width: 500px) {
```
Comment (SauveJeanLuc) [inline frontend/src/components/Services/services.scss:204]: What about nesting these 😉?... into something like:

```
@media (max-width: 500px) {
    .services .services-section-2 .service-1 {
        width: 111px;
        font-size: 12px;
        height: 120.5px;
    }

    .services .services-section-2 .service-2 {
        font-size: 14px;
        line-height: 11px;
        width: 163.5px;
        height: 158.5px;    
    }
    .services .services-section-2 .service-2 h2 {
        margin: 5px;
        font-size: 13px;
    }
    .services .services-section-2 .service-2 p{
        font-size: 14px;
        margin: 10px 5px;
    }
    .services .services-section-2 .service-3 {
        width: 111px;
        font-size: 12px;
        height: 120.5px;  
    }
    .services .services-section-2 img {
        display: none;
    }
}
```
Btw, you're killin' it @Cyebukayire 🚀🤘🏾.

**Comment by andrewtlu** [comment]
See the following paste for my terminal output: https://pastebin.com/jveejbbE
I am running `npm start` from the frontend repository, and it seems like the `/src/components/navbar` may need to be renamed to `/src/components/Navbar`.
I'm not sure about the other errors, but take a look and let me know.

**Comment by Cyebukayire** [comment]
Thank you so much, Andrew:)
The rest warnings are fixed.
Let me update the README file. (`npm i` will automatically install all added dependencies.)

**Pull request merged**